contracts/std-reference/src/contract.rs

```diff
@@ -157,75 +183,59 @@ fn execute_force_relay(
 #[cfg_attr(not(feature = "library"), entry_point)]
 pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
     match msg {
-        QueryMsg::Config {} => to_binary(&query_config(deps)?),
-        QueryMsg::IsRelayer { relayer } => to_binary(&query_is_relayer(deps, &relayer)?),
+        QueryMsg::Admin {} => to_binary(&ADMIN.query_admin(deps)?),
+        QueryMsg::IsRelayer { relayer } => {
+            to_binary(&query_is_relayer(deps, &deps.api.addr_validate(&relayer)?)?)
+        }
         QueryMsg::GetRef { symbol } => to_binary(&query_ref(deps, &symbol)?),
-        QueryMsg::GetReferenceData {
-            base_symbol,
-            quote_symbol,
-        } => to_binary(&query_reference_data(deps, &base_symbol, &quote_symbol)?),
-        QueryMsg::GetReferenceDataBulk {
-            base_symbols,
-            quote_symbols,
-        } => to_binary(&query_reference_data_bulk(
-            deps,
-            base_symbols,
-            quote_symbols,
-        )?),
+        QueryMsg::GetReferenceData { symbol_pair } => {
+            to_binary(&query_reference_data(deps, &symbol_pair)?)
+        }
+        QueryMsg::GetReferenceDataBulk { symbol_pairs } => {
+            to_binary(&query_reference_data_bulk(deps, &symbol_pairs)?)
+        }
     }
 }
 
-fn query_config(deps: Deps) -> StdResult<Config> {
-    CONFIG.load(deps.storage)
-}
-
 fn query_is_relayer(deps: Deps, relayer: &Addr) -> StdResult<bool> {
     Ok(RELAYERS.may_load(deps.storage, relayer)?.is_some())
 }
 
 fn query_ref(deps: Deps, symbol: &str) -> StdResult<RefData> {
     if symbol == "USD" {
-        return Ok(RefData::new(Uint128::new(E9), u64::MAX, 0));
+        return Ok(RefData::new(E9, Uint64::MAX, Uint64::zero()));
```

Comment:
as it's not the last line.  I think the return style should be the same for both branches (with or without return).
I prefer no return for both in this case.